src/edu/washington/escience/myria/MyriaConstants.java

```diff
@@ -68,6 +68,11 @@
   public static final String EXEC_ENV_VAR_FRAGMENT_RESOURCE_MANAGER = "fragmentResourceManager";
 
   /**
+   * Test mode is true
+   */
+  public static final String EXEC_ENV_VAR_TEST_MODE = "testMode";
```

Comment:
Could you say a little more about this test mode? Why `pyFuncRegistrar` can't be set in test mode?